### ISSUE TITLE
publish to crates.io

### ISSUE THREAD

**Issue by xlc** [paritytech/try-runtime-cli#93]

**Comment by AlexD10S** [comment]
Is there any plan to publish on crates.io? Publishing would be ideal as it would allow external tools to integrate this functionality seamlessly.

**Comment by kianenigma** [comment]
@rockbmb is intending to maintan this repo after his current duties are over. I think this is a good item to pay attention to once he has some bandwidth. 

Doing the publishing is actually not particularly time-consuming, it can be done sooner. 

cc @EgorPopelyaev @ggwpez

**Comment by rockbmb** [comment]
@kianenigma thanks for the ping.

My hands are tied with the Asset Hub migration at the moment, but yes, I'm looking to look after this in the future.

**Comment by AlexD10S** [comment]
> Doing the publishing is actually not particularly time-consuming, it can be done sooner.
> 
> cc [@EgorPopelyaev](https://github.com/EgorPopelyaev) [@ggwpez](https://github.com/ggwpez)

Thanks, Kian! I believe no changes are required for publishing the `try-runtime-core` crate, but I’m happy to help if any changes are needed. Integrating this functionality into [pop-cli](https://github.com/r0gue-io/pop-cli) would be very helpful for us.

**Comment by AlexD10S** [comment]
Just wanted to kindly check in and see if there have been any updates on this issue.

**Comment by ggwpez** [comment]
No, this repo is still using git dependencies which prevents publishing. Currently in limbo with possibly no maintainer.

**Comment by AlexD10S** [comment]
Thanks for the update @ggwpez .

We've just integrated this functionality into our tool, [Pop CLI](https://github.com/r0gue-io/pop-cli). At the moment, we're fetching the `try-runtime-cli` binaries and wrapping the functionality ourselves, but it would be much cleaner, and more powerful, if we could import the crate directly.

Let us know if we can support in any way to help keep the tool up to date!

**Comment by kianenigma** [comment]
I tried bringing this to a stable release in a branch and most things compile. We will not say no to a PR that brings this up to date + prepare it for a release in the meantime :) cc @ggwpez @rockbmb

**Comment by ggwpez** [comment]
So we are not using git-dependencies anymore? Nice  

What else would you change? I would just bump the version number + CHANGELOG, we did not fix any of the reported bugs anyway.

**Comment by rockbmb** [comment]
This will have to wait a while longer.

`v0.9.0`, created with #122, requires some features that don't exist in any published version of Polkadot SDK; thus, dependencies are still pinned to git revisions, and publication to crates.io forbids this.
